pkgs/http/test/html/cache_test.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@TestOn('browser')
+library;
+
+import 'dart:async';
+
+import 'package:http/browser_client.dart';
+import 'package:http/http.dart' as http;
+import 'package:http/src/exception.dart';
+import 'package:test/test.dart';
+
+import 'utils.dart';
+
+void main() {
+  test('#send a StreamedRequest with default type', () async {
```

Comment:
Do any of these tests require streaming? If you look at the cronet tests:
https://github.com/dart-lang/http/blob/master/pkgs/cronet_http/example/integration_test/cronet_configuration_test.dart

You'll see that they are confirming that the server receives the expected number of requests.

Reply:
the server is IO-based not web based. so, we can try to manuever those tests but adding retry parameter in the test() function.